package.json

```diff
@@ -145,7 +145,8 @@
           {"type": "docs", "scope":"README", "release": "patch"},
           {"type": "chore", "release": "patch"},
           {"type": "refactor", "release": "patch"},
-          {"type": "style", "release": "patch"}
+          {"type": "style", "release": "patch"},
+          {"type": "fix", "release": "patch"}
```

Comment:
`fix` is already part of the `"preset": "angular"` definition.. I'd suggest either removing the `.releaserc` file or removing the `"release"` field from the package.json since it was an experiment for the last release anyway